src/test/java/taskFinal/util/Address.java

```diff
@@ -0,0 +1,53 @@
+package taskFinal.util;
+
+import com.github.javafaker.Faker;
+
+import java.util.Objects;
+
+public class Address {
+    private static String cell;
+    private static String street;
+    private static String city;
+    private static String state;
+    private static String zip;
```

Comment:
It is a bit unclear why do you need to make them static. Could you please explain ?

Reply:
When I was writing method: public static Address getDefaultAddress(), Intellij IDEA warned, that static method is operating only with static variables